admin_guide/scheduling/scheduler.adoc

```diff
@@ -97,162 +307,345 @@ and a string match with the name of the host.
 {"name" : "HostName"}
 ----
 
+**_NoVolumeNodeConflict_** 
+----
+{"name" : "NoVolumeNodeConflict"}
+----
+
+[[admin-guide-scheduler-general-predicates]]
+=== General Predicates
```

Comment:
Not sure if by `General Predicates` here we mean in some literal sense? Otherwise, in scheduler, following are general predicates (divided into 2 categories, non-critical and essential): 
non-critical: PodFitsResources
essentials: PodFitsHostPorts, HostName, MatchNodeSelector